iphone/cli/commands/_build.js

```diff
@@ -1810,7 +1810,7 @@ iOSBuilder.prototype.validate = function validate(logger, config, cli) {
 		if (cli.argv['source-maps']) {
 			this.sourceMaps = true;
 			// if they haven't, respect the tiapp.xml value if set one way or the other
-		} else if (cli.tiapp.hasOwnProperty['source-maps']) { // they've explicitly set a value in tiapp.xml
+		} else if (cli.tiapp.hasOwnProperty('source-maps')) { // they've explicitly set a value in tiapp.xml
```

Comment:
<!--
  1 failure:  iphone/cli/comman...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: iphone/cli/commands/_build.js;
  Line: 1813;
-->
- :no_entry_sign: iphone/cli/commands/_build.js line 1813 – Do not access Object.prototype method 'hasOwnProperty' from target object. (no-prototype-builtins)